src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,11 +1,41 @@
-export const TodoFilter = () => (
+import { FC } from 'react';
+
+enum SelectOption {
+  All,
+  Active,
+  Completed,
+}
```

Comment:
And you should use it in App 
`const [filter, setFilter] = useState<Filter>(Filter.All);`

```suggestion
enum Filter {
  All = 'all,
  Active= 'active',
  Completed= 'completed',
}
```